src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -31992,21 +31992,21 @@ public static int TakeOutParam<T>(T y, out T x)
                 var compilation = CreateCompilationWithMscorlib45(source, options: TestOptions.ReleaseExe, parseOptions: TestOptions.Regular9);
 
                 compilation.VerifyDiagnostics(
-                    // (3,12): error CS0116: A namespace cannot directly contain members such as fields or methods
+                    // (3,12): error CS9010: A top-level entry point cannot directly contain members such as fields, properties, and methods
```

Comment:
>A top-level entry point cannot directly contain members such as fields, properties, and methods [](http://example.com/codeflow?start=45&length=95)

My preference would be to not do anything, or to simply adjust the wording for the original error to something like: "A namespace **or a top-level entry point** cannot directly contain members such as fields or methods"

Reply:
@AlekseyTs Adjusting the existing message sounds reasonable, but won't play nicely with @333fred's suggestion: https://github.com/dotnet/roslyn/pull/59321#discussion_r806361327

Reply:
> Adjusting the existing message sounds reasonable, but won't play nicely with @333fred's suggestion

I am not sure we need to worry about that because the message clearly talks about **members**. Local functions are not members. Alternatively we can replace methods with properties: "A namespace **or a top-level entry point** cannot directly contain members such as fields or **properties**, etc." Or we could say ordinary methods: "A namespace **or a top-level entry point** cannot directly contain members such as fields or **ordinary** methods, etc."

Bottom line, I prefer us not touching implementation.